include/alpaka/test/KernelExecutionFixture.hpp

```diff
@@ -29,36 +29,38 @@ namespace alpaka::test
         using Acc = TAcc;
         using Dim = alpaka::Dim<Acc>;
         using Idx = alpaka::Idx<Acc>;
-        using PlatformAcc = Pltf<Acc>;
-        using DevAcc = Dev<Acc>;
-        using PltfAcc = Pltf<DevAcc>;
-        using QueueAcc = test::DefaultQueue<DevAcc>;
+        using Platform = alpaka::Pltf<Acc>;
+        using Device = alpaka::Dev<Acc>;
+        using Queue = test::DefaultQueue<Device>;
         using WorkDiv = WorkDivMembers<Dim, Idx>;
 
         KernelExecutionFixture(WorkDiv workDiv)
-            : m_devHost(getDevByIdx(m_platformHost, 0))
-            , m_devAcc(getDevByIdx(m_platformAcc, 0))
-            , m_queue(m_devAcc)
-            , m_workDiv(std::move(workDiv))
+            : m_devHost{getDevByIdx(m_platformHost, 0)}
+            , m_device{getDevByIdx(m_platform, 0)}
+            , m_queue{m_device}
+            , m_workDiv{std::move(workDiv)}
         {
         }
 
         template<typename TExtent>
         KernelExecutionFixture(TExtent const& extent)
-            : KernelExecutionFixture(getValidWorkDiv<Acc>(
-                getDevByIdx(m_platformAcc, 0),
-                extent,
-                Vec<Dim, Idx>::ones(),
-                false,
-                GridBlockExtentSubDivRestrictions::Unrestricted))
+            : m_devHost{getDevByIdx(m_platformHost, 0)}
+            , m_device{getDevByIdx(m_platform, 0)}
+            , m_queue{m_device}
+            , m_workDiv{getValidWorkDiv<Acc>(
+                  m_device,
+                  extent,
+                  Vec<Dim, Idx>::ones(),
+                  false,
+                  GridBlockExtentSubDivRestrictions::Unrestricted)}
```

Comment:
This is the bugfix. `: KernelExecutionFixture(getValidWorkDiv<Acc>(getDevByIdx(m_platformAcc, 0), ...), ...)` uses `m_platformAcc` but the constructor of the object has not run yet. And this only occured in the SYCL backend, because it is the only stateful backend. Thanks for spotting this!